labs-ios-starter/Map/MapViewController.swift

```diff
@@ -33,7 +35,9 @@ class MapViewController: UIViewController, MKMapViewDelegate {
         }
         
         mapView.delegate = self
+        addUserTrackingButton()
         addSearbarTable()
+        
     }
     
     func addSearbarTable() {
```

Comment:
not a spelling contest, but search is misspelled

Reply:
Thank you, ill definitely fix that next PR